checker/manual/lock-checker.tex

```diff
@@ -872,6 +872,54 @@ \subsection{Discussion of \<@Holding>\label{lock-checker-holding}}
 \end{itemize}
 
 
+\section{Other lock annotations\label{lock-other-annotations}}
+
+The Checker Framework's lock annotations are similar to annotations used
+elsewhere.
+
+If your code is already annotated with a different lock
+annotation, you can reuse that effort.  The Checker Framework comes with
+cleanroom re-implementations of annotations from other tools.  It treats
```

Comment:
If so, then the current versions need to be deleted and re-implemented, for copyright/licensing issues.